src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,59 @@
+package duke.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected TaskType taskType;
+    enum TaskType {
```

Comment:
Is this Enum needed? 
Also if needed it is better to declare it outside this class i.e. in a separate file

Reply:
Only implemented the enum to fulfil the 'A-Enum' optional so I think I shall leave it in for now~ Will follow your suggestion to declare it outside the class~!